packages/report-flaky-tests/lib/github-api.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import * as github from '@actions/github';
+import * as unzipper from 'unzipper';
+
+/**
+ * Internal dependencies
+ */
+import type { FlakyTestResult } from './types';
+
+class GitHubAPI {
+	octokit: ReturnType< typeof github.getOctokit >;
```

Comment:
Can we just type this as `octokit: Octokit`? I think the helpers (`ReturnType`, `typeof`) are neat but they force the reader of the code to have to think for a second which can add up.